tensorflow/compiler/tf2tensorrt/convert/convert_nodes.h

```diff
@@ -533,7 +533,7 @@ class Converter {
   // in the list is numbered according to TRT convention (see ConvertAxis for
   // details).
   Status SqueezeTensor(nvinfer1::ITensor* input,
-                       const std::vector<int>& trt_axes,
```

Comment:
Need to fix the document for the parameter.

Reply:
updated